receiver/k8sobjectsreceiver/README.md

```diff
@@ -36,9 +36,32 @@ The following is example configuration
         mode: watch
         group: events.k8s.io
         namespaces: [default]
+
+  k8sobjects/leader-election:
+    leader_election:
+      enabled: true
+      lock_name: "k8sobjects-leader-election"
+    auth_type: serviceAccount
+    objects:
+      - name: pods
+        mode: pull
+        label_selector: environment in (production),tier in (frontend)
+        field_selector: status.phase=Running
+        interval: 15m
+      - name: events
+        mode: watch
+        group: events.k8s.io
+        namespaces: [default]
 ```
 
 Brief description of configuration properties:
+- `leader_election`: Leader election mode helps to ensure that only one instance (as leader instance) is collecting `k8sobjects` data, and if the leader instance fails, another one is elected as leader and takes its place. 
+*Note*: You need to add additional RBAC to perform the leader election mode. See below [Leader Election RBAC](#RBAC-Leader-Election) for more information.
+  - `enabled` (default = `false`): whether run in leader election mode.
+  - `lock_name` (default = `k8sobjects`): the identity name of holder, will use component's ID if not set.
+  - `lease_duration` (default = `15s`): the duration that non-leader candidates will wait to force acquire leadership.
+  - `renew_deadline` (default = `10s`): the duration that the acting master will retry refreshing leadership before giving up.
+  - `retry_period` (default = `2s`): the duration the LeaderElector clients should wait between tries of actions. 
```

Comment:
I'm not sure what this means just by reading this doc. Are "LeaderElector clients" referred to as "non-leader candidates" in a previous item?